.github/workflows/ibis-backends.yml

```diff
@@ -563,6 +563,7 @@ jobs:
               - "pyspark@3.5"
               - "'pandas@>2'"
               - "'numpy@>1.24'"
+              - "delta-spark"
```

Comment:
so wasn't testing because we didn't have this in CI, so it was import-or-skipping?

Reply:
Yep!